app/helpers/evaluations_helper.rb

```diff
@@ -10,13 +10,14 @@ def sum_work_details(point_details, user)
   end
 
   def sum_responsibility_details(point_details, user)
-    [sum_details(point_details, user, Principle::RESPONSIBILITY), 20].min
+    [sum_details(point_details, user, Principle::RESPONSIBILITY), 50].min
   end
 
   def sum_all_details(point_details, user)
     sum = sum_responsibility_details(point_details, user) + sum_work_details(point_details, user)
-    return 5 if [3, 4].include? sum
-    return 0 if [1, 2].include? sum
+    sum_clipped = [sum, 50].min
```

Comment:
What do you think if we clip the sum at the [last line](https://github.com/kir-dev/pek-next/pull/109/files#diff-6905bd267be21b0c2a6ce80da28e260cR22) of this method? It won't affect the next two statement, probably saves some performance, but the most important, it makes the method more expressive.